test/e2e/interest-groups/index.test.js

```diff
@@ -28,10 +30,13 @@ describe('Fledge', () => {
 				bidding_logic_url: 'http://localhost:3000/test/e2e/mock/bl.js',
 			};
 			const expiry = 60000;
-			await page.evaluate((igObject, expiry) => window.fledge.joinAdInterestGroup(igObject, expiry), igObject, expiry);
+			await page.evaluate((igObject, expiry) => {
+				const fledge = new window.fledge();
+				return fledge.joinAdInterestGroup(igObject, expiry);
+			}, igObject, expiry);
 			const result = await page.evaluate(() => {
 				// eslint-disable-next-line
-				const myPromise = new Promise((resolve, reject) => {
+				const myPromise = new Promise((resolve) => {
```

Comment:
```suggestion
				return new Promise((resolve) => {
```

Reply:
you'll have to remove the return on line 44 https://github.com/MagniteEngineering/fledge.polyfill/pull/66/files#diff-94624f5db75aaf54292f83c6e1d277db87ec1a892ec83bab6efb3bf8c62a59a2R49